calculator.html

```diff
@@ -5,7 +5,6 @@
   <meta charset="utf-8">
   <title>Calculator</title>
   <link rel="stylesheet" type="text/css" href="calculator.css">
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/knockout/3.4.0/knockout-min.js"></script>

```

Comment:
We are no longer referencing knockout from a CDN. We could still do this with RequireJS but I have chosen not to for now.